tests/blockchain/eth/contracts/main/staking_escrow/test_staking_escrow_additional.py

```diff
@@ -218,23 +221,23 @@ def test_re_stake(testerchain, token, escrow_contract):
     testerchain.wait_for_receipt(tx)
 
     # Set re-stake parameter even before initialization
-    _wind_down, re_stake, _measure_work = escrow.functions.getFlags(staker).call()
+    _wind_down, re_stake, _measure_work, _snapshots = escrow.functions.getFlags(staker).call()
     assert re_stake
     tx = escrow.functions.setReStake(False).transact({'from': staker})
     testerchain.wait_for_receipt(tx)
-    _wind_down, re_stake, _measure_work = escrow.functions.getFlags(staker).call()
+    _wind_down, re_stake, _measure_work, _snapshots = escrow.functions.getFlags(staker).call()
```

Comment:
For these flags, given that they are bits, if we haven't already we should do a post check that the other flags remain unchanged after setting one of them. Doesn't have to be here specifically, and doesn't have to be in every test, but somewhere nonetheless.

Reply:
✅ Added `tests.blockchain.eth.contracts.main.staking_escrow.test_staking_escrow_additional.test_flags`